### PR TITLE
[Callout][HTML] Accordion + Feedback

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

New Callout accordion + Callout Feedback List HTML components

-----

image
image
image
image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8085/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8092/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8096/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8103/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

File: packages/scss/src/components/calloutDisclosure/component.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	a,
+	.link {
+		--commons-text-link-color: var(--palettes-grey-800);
+		--commons-text-link-hover: var(--palettes-grey-700);
+	}
+
+	@at-root ($atRoot) {
+		.calloutDisclosure-summary {
+			display: flex;
+			gap: 0.75rem;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/calloutDisclosure/component.scss:22]: `.75rem` seems to happen more and more. Maybe we should include it in our spacings ?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8126/ui-diff)

File: packages/scss/src/components/calloutDisclosure/component.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+				@include a11y.focusVisible($offset: 3px, $borderRadius: 6px);
+			}
+		}
+
+		.calloutDisclosure-summary-icon {
+			--icon-size: 1.5rem;
+			color: var(--palettes-700, var(--palettes-grey-700));
+		}
+
+		.calloutDisclosure-summary-title {
```
Comment (BertrandPodevin) [inline packages/scss/src/components/calloutDisclosure/component.scss:36]: Seems an unlikely case, but long non-secable title (eg aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa) may trigger an overflow

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8128/ui-diff)

**Comment by BertrandPodevin** [comment]
Maybe add a small animation when openning ? Basics ones could be done with pure CSS

https://codepen.io/bertrandpodevin/pen/rNPOaqQ

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8129/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8133/ui-diff)

File: packages/scss/src/components/calloutDisclosure/component.scss (hunk 1 above)
Comment (jeremie-lucca) [reply packages/scss/src/components/calloutDisclosure/component.scss:22]: @BertrandPodevin Yep, it's planned. But to do it we have to shift multiple vars which might be painful. So we wait tokens implementation to cover it with extra values like `.375` and maybe `.125`

**Comment by jeremie-lucca** [comment]
@BertrandPodevin Imho it's cleaner without animation, but I'll ask Clément.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8194/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2378/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8298/ui-diff)

**Pull request closed without merging**